RecoLocalCalo/HGCalRecProducers/plugins/HGCalRecHitWorkerSimple.cc

```diff
@@ -99,7 +99,7 @@ bool HGCalRecHitWorkerSimple::run(const edm::Event & evt, const HGCUncalibratedR
         hid = detid;
         thickness = ddds_[hid.subdetId() - 3]->waferTypeL(hid.wafer());
         sigmaNoiseGeV = 1e-3 * weights_[layer] * rcorr_[thickness]
-                    * hgcHEF_noise_fC_[thickness - 1] / hgcHEF_fCPerMIP_[thickness - 1];
+                    * hgcEE_noise_fC_[thickness - 1] / hgcEE_fCPerMIP_[thickness - 1];
```

Comment:
This is just a bug fix for a typo, I would say...

By the way, I think you should have a "break" also after next L111, unless you want to treat "case HcalEndcap:" and "case HGCHEB:" the same: please check